File: backend/src/main/java/com/getyourtour/Controller.java
package com.getyourtour;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class Controller {
    @GetMapping("")
    public String index() {
        return "Get Your App";
    }
}

File: backend/src/main/kotlin/com/getyourtours/Controller.kt
package com.getyourtours

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class Controller {

    @RequestMapping("/")
    fun index(): String {
        return "Welcome to main page of GetYourTours :D"
    }

    @RequestMapping("/about")
    fun about(): String {
        return "This is a app for tour reservation management!"
    }
}